dna/zomes/projects/src/project/goal_vote/validate.rs

```diff
@@ -35,6 +37,8 @@ pub fn validate_create_entry_goal_vote(
 }
 
 #[hdk_extern]
+/// Updates allowed if Goal exists and only by the original author
```

Comment:
`must_get_header_and_entry::<GoalVote>`

so it's the original GoalVote that must exist not the Goal... we assume the Goal existed if the validation for create had passed...

Reply:
```suggestion
/// Updates allowed if the original `GoalVote` exists and only by the original author
```